datasets/us/dod_chinese_milcorps/crawler.py

```diff
@@ -0,0 +1,26 @@
+from zavod import Context
+import csv
+
+def crawl(context: Context):
+    source_file = context.fetch_resource("source.csv", context.dataset.data.url)
+    with open(source_file, "r") as f:
+        reader = csv.DictReader(f)
+        for row in reader:
+            entity = context.make("Company")
+            name = row.pop("Name")
+            entity.id = context.make_id(name)
+            entity.add("name", name)
+            entity.add("alias", row.pop("Alias", None))
+            clean_name = row.pop("Clean Name", None)
+            if clean_name != "" and clean_name != name:
+                entity.add("alias", clean_name)
+            entity.add("notes", row.pop("Note", None))
+            parent_name = row.pop("Parent Name", None)
+            if parent_name != "" and parent_name != name:
+                parent = context.make("Company")
+                parent.id = context.make_id(parent_name)
+                parent.add("name", parent_name)
+                context.emit(parent)
```

Comment:
Do I need to set `external=True` here?

Reply:
do you mean `target`? Nope - only the directly-listed entities are [targets](https://www.opensanctions.org/docs/glossary/#targets)